docs/RenderEngine/Textures/Text.md

```diff
@@ -11,7 +11,7 @@ You can use various properties to control the way in which you want to render te
 | Name | Type | Default | Description |
 |---|---|---|---|
 | `text` | String | '' | Text to display |
-| `fontFace` | Integer | null | Font family used for current text |
+| `fontFace` | String or String[] | null | Font family used for current text. If an array is provided, font familes that appear later in the array will be used as fallbacks. |

```

Comment:
familes => families

also about the default, is that really `null` in practice? there is some kind of final fallback font if nothing is specified - maybe we should make some mention of that

Reply:
Yeah looking at it deeper there's a lot more detail we can add here, so I detailed it out a good bit more.